sample/src/main/java/com/metaplex/sample/FirstFragment.kt

```diff
@@ -68,13 +78,19 @@ class FirstFragment : Fragment() {
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        (activity as AppCompatActivity?)!!.supportActionBar!!.setIcon(R.drawable.ic_metaplex_logo_mark)
```

Comment:
@thedevyansh we should never be using assertions in the app like this, because failed assertions will crash the app. The assertion operation (`!!`) should only be used in testing, or when you have first verified that its usage is safe so that it will not crash the app. I recommend that you use the null coalescing operator instead (`?.` or `?:`).

Reply:
Will update it. Thanks!